### PR TITLE
Minor style fixes

### PR DESCRIPTION
**Pull request by djyou**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py [hunk 1]
```diff
@@ -26,21 +26,20 @@
...
-                        resource_group_name=None):
+BUILD_NOT_SUPPORTED = 'Builds are only supported for managed registries.'
 
-    resource_group_name = get_resource_group_name_by_registry_name(
-        cmd.cli_ctx, registry_name, resource_group_name)
 
+def acr_build_show_logs(client,
+                        build_id,
+                        registry_name,
+                        resource_group_name):
```
Comment (ankurkhemani) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py:42]: Why did we make `resource_group_name` required argument?

Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py:42]: This is an internal method and the caller already has `resource_group_name` resolved. Unnecessary overhead here.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py [hunk 2]
```diff
@@ -288,38 +278,35 @@ def acr_build(cmd,
...
-    else:
-        build_id = queued_build.build_id
-        print("Queued a build with ID: {0}".format(build_id))
-        print("Waiting for a build agent...")
-        acr_build_show_logs(cmd, client, registry_name, build_id, resource_group_name)
+
+    build_id = queued_build.build_id
+    print("Queued a build with build ID: {}".format(build_id))
+    print("Waiting for a build agent...")
+    return acr_build_show_logs(client, build_id, registry_name, resource_group_name)
```
Comment (ankurkhemani) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py:309]: nit: return is not needed here.

**Review by ankurkhemani — APPROVED** [review]

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py (hunk 2 above)
Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py:309]: Just to make pylint happy. We also had return in `build_task.py`.

**Pull request merged**